bootstrap-env.sh

```diff
@@ -63,7 +63,7 @@ function info
 
 function uberenv
 {
-    python scripts/uberenv/uberenv.py "$ALL_ARGS"
+    python scripts/uberenv/uberenv.py "$@"
```

Comment:
This is the proper way to forward args. Not sure why the other method did not work